src/check_imap.sh

```diff
@@ -10,5 +10,5 @@ fi
 
 JSON="${RUN_DIR}/lib/JSON.sh"
 
-MATCHING=$(${JSON} < "${IMAP_SOURCE}" | grep -v "\[\]" | grep "$1")
+MATCHING=$(${JSON} < "${IMAP_SOURCE}" 2>/dev/null| grep -v "\[\]" | grep "$1")
```

Comment:
maybe generating a tmpfile log is a better solution for testing later

Reply:
What do you mean? Like make this a variable that can be set in config?

Reply:
Like instead of redirecting stderr to /dev/null we could redirect all error messages to a logfile associated with the last run, a temporary file that has a ttl of like 1 day. Then we don't ignore this stuff and can look into what has gone wrong if anything ever does

Reply:
For tests specifically it should be a tmp file with a lifespan of the tests lifespan though